update non-breaking deps

zarsky-broad opened the pull request:
Created while waiting for Jenkins runs to pass last week...

zarsky-broad committed:
update non-breaking deps

zarsky-broad commented on project/Dependencies.scala:
```diff
@@ -62,24 +62,24 @@ object Dependencies {
+    "org.webjars.npm"                % "swagger-ui-dist"     % "3.45.0",
     "org.webjars"                    % "webjars-locator"     % "0.40",
-    "com.pauldijou"                 %% "jwt-core"            % "3.1.0",
+    "com.github.jwt-scala"          %% "jwt-core"            % "7.1.1",
```
This is one thing I think might benefit from further testing

davidangb replied:
Cloud Integrations uses JWT, definitely check with them on this one

davidangb reviewed (commented):
Some of the dependency changes upgrade major versions of libraries. Have you done the due diligence of reading changelogs or other research to know if there is any functional impact?

davidangb commented on project/build.properties:
```diff
@@ -1 +1 @@
-sbt.version=1.4.6
+sbt.version=1.4.9
```
ideally this would also have a corresponding change to the baseimage used for building

zarsky-broad committed:
PR comments

zarsky-broad commented:
Notable updates from changelogs of dependencies:
1. `akka-http` - support for new things added, but no notable changes to existing features.
2. `jackson` - support for new things added, but no notable changes to existing features.
3. `netty-codec` - bugfixes
4. `lucene-queryparser` - apparently almost no changes, like 1 bugfix
5. `guava` - minor bugfixes and a bunch of new methods, no notable changes; switched from the android build to the jre one, which should be a bit smaller, but we shouldn't see any difference since we're not building for android or running on java 7
6. `log4j` - support for new things added, but no notable changes to existing features.
7. `logback-classic` - bugfixes
8. `scala-logging` - very minor bugfixes
9. `parboiled-core` - internal upgrades only
10. `google-auth-library-owuth2-http` - bugfixes, new features, one breaking change to something we don't use
11. `swagger-ui` - some new features and ux improvements, nothing noteworthy
12. `jwt-scala` - nothing at all relevant to our usage
13. `univocity-parsers` - improvements to parsing efficiency that we might want to look into *(including support for iterables!)*, but no breaking changes
14. `prettytime` - only used by one function that wasn't actually called from anywhere, so safe to remove!
15. `json-schema` - no breaking changes
16. `better-files` - nothing breaking, *but support for having temp files delete themselves when they've been used, which we might not be doing now?*
17. `httpclient` - not used directly, removed
18. `specs2-core` - unused, removed
19. `scalatest` - some breaking changes, used provided codemod tool
20. `mockserver-netty` - minor breaking renames, accounted for

ansingh7115 reviewed (approved):
Awesome! Thank you for cleaning this up!
